### PR TITLE
Removes heretics gamemode, they can only spawn on dynamic now.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Sets the probability of heretics to 0 on both sage and golden.

## Why It's Good For The Game

bla bla bla big problem is no adaptability:
 - 1 goal (kill) and only provided with 1 way to do it.

Can still spawn on dynamic, but will be FAR less frequent now. Heretics on their own as a gamemode are pretty awful.
Heretics are a TG gamemode by blood and absolutely do not work on a server that isn't trying to encourage murderboning and validhunting.

Heretics are extremely poorly designed and have murderboning as one of the core parts of their antag.
The biggest difference between traitor / changeling killings and heretic killings is that the objectives traitors and changelings get require you to kill 1 person. Having >1 hour to kill 1 person allows you to plan, gather intel on your target and take them out in a stealthy manner if you wish leading to deaths later on in the round and allows for more creative stuff like traps.
Heretics are required to kill a lot of people to succeed. (Between 2 and 8 people, on average 5.3 people). This means that instead of having 1 hour to kill a target, you have close to 15 minutes per target. The only real effective strategy as a heretic is to rush and outright murder your target which immediately outs you as a heretic and you have the entirity of security rushing you with lethals.

The only option for heretics once they get going is to kill and avoid interacting with the rest of the game completely. Traitors primarilly rely on stealth, doing parts of your job to not araise suspicious while sneaking off every now and again to complete your real missions. This allows you to continue to interact with other people when you aren't doing killings. Traitors primary focus is generally covering your tracks and trying to complete your objectives without being noticed. Even on hijack one of the best strategies is to lay low and pick people off, or take the shuttle without killing too many people as trying to straight murderbone usually results in death.
Another antagonist that lethals are essentially required for are changelings. Why are changelings better than heretics when they both get instantly lethalled?
Changelings sole function is to be an antag that excells at deception. Interacting with other players is fundamental when you are disguised as someone else and trying to decieve them.
Heretics on the other hand are a kill, get spotted immediately, get gear and repeat. Changelings can always disguise and evade capture if spotted, whereas heretics are forced into continuing their killings.

Anyway that was some objective based problems, solo antag killing like 4-8 people obviously is stupid.

Anyway what makes a good antag imo?
Adaptability and the ability to approach a problem in multiple ways.

Traitors and changelings accomplish this by having a variety of different gear / abilities which allow for the same problem to be approached in many different ways.
heretics have only 1 problem: The problem of killing people (Their only objectives are to kill people and stand near someone for 5 minutes) and while they have 3 different routes they can take which offers different abilities, the approach provided by each ability is not unique and the only way to approach a problem is a head on attack. There are items that allow you to blind your target, but considering the amount of things that you need to find which are already hard to come across (eyes, heart and something literally held by your target) in basically all cases its better to just murder your target rather than try to convince them to give you their gloves so you can disable them a bit before you murder them.

Basically antag sucks, im not reworking this apparently someone else was working on a rework according to #general so they can reimplement heretics when they make it <s>not complete shit</s> slightly different. I'd rather work on an entire new antag than fix one that is flawed at the core.

## Changelog
:cl:
refactor: Heretics can now only spawn on dynamic.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Also they are countered by null rod and chaplains items but the chaplain can be a heretic which kind of defeats the point

**Comment by PowerfulBacon** [comment]
And also the path that allows converting people to your minions has no way to deconvert the minions apart from killing and cloning them which seems stupid

**Comment by PowerfulBacon** [comment]
Also being instantly outed by a peirced reality is dumb

**Comment by PowerfulBacon** [comment]
Also a lot of the stuff in heretics is extremely poorly communicated / makes barely any sense like mansus fist being used on the ground to draw runes

**Comment by PowerfulBacon** [comment]
This is a nitpick but if anyone actually wants stuff to change in heretics, the invocation text for the spells are extremely uncreative and usually just the word with the middle character set as '

**Comment by PowerfulBacon** [comment]
Murderbone rules are also incredibly blured, heretics main point is to ascend but
1. Is the process of sacraficing people after objectives done murderboning and allowed?
2. Is murderboning allowed once ascended and you are put in a form made to murderbone

**Comment by PowerfulBacon** [comment]
Whats the point of items that favour stealth like a thermal vision amulate when wearing the amulate is a straight giveaway?

**Comment by PowerfulBacon** [comment]
Another thing about chaplain being a heretic, if the chaplain makes and blesses things why are they blocked from using spells by their own holyness. (Ashen jaunt blocked by blessed turfs).
Also the fact that the only person who can counter heretic spells can be a heretic is pretty dumb

**Comment by PowerfulBacon** [comment]
Also I'd like to make an edit and say that I'm wrong about no way to deconvert the flesh monsters.
Apparently on death the flesh monsters lose their allegience to their master and regain normal max health, however they still remain a ghoul and keep the positive traits such as stun immunity and the heretic faction

**Comment by eeSPee** [comment]
The problem wasn't adaptability; heretics have a nice arsenal of stuff, and that traps/elixirs will be amazing when ported. The problem is that it's hidden behind a tech tree that is hard to acquire and it's slow to progress. And the only ways to gain resource is to out all heretics or murderbone.

**Comment by Sinestia** [comment]
> 
> 
> The problem wasn't adaptability; heretics have a nice arsenal of stuff, and that traps/elixirs will be amazing when ported. The problem is that it's hidden behind a tech tree that is hard to acquire and it's slow to progress. And the only ways to gain resource is to out all heretics or murderbone.

I'm not really sure if further encouraging of non-antagonism is a great solution


I'm not sure why /tg/ players manage to do this a lot more competently without as much handholding, ngl lol.

 Unironically, consider turning on respawn to ease the "killing problem" if people are so afraid to do it outside of the last 5 minutes of a round. 
(unfortunately that would probably require tools to make it easier for admins and players to figure out if they've been meta'd)

**Comment by eeSPee** [comment]
> I'm not sure why /tg/ players manage to do this a lot more competently without as much handholding, ngl lol.

They don't, lol. Heretics on TG have the same problem as here; it's impossible to stealth and easy as fuck to meta.

They just murderbone, and have the void path which is better than any other paths.

**Comment by Kapu1178** [comment]
based. Fuck heretics. Non-stop heretic complaints from both Golden and Sage players. Shitty gamemode.

**Comment by FriendlyContractor** [comment]
Well i personally dont have a problem with heretic objectives. I consider dying to be a natural part of the game and enjoy their aggresive playstyle. I have no idea about how others in community feel about heretics and would like to see a pool about how they feel about them being removed as well as their opinion on heretics in general. Anyway - that was my 5 cents. 

Ya all have a nice day

**Review by ike709 — DISMISSED** [review]
Apply the config changes for me and I'll merge

**Review by Crossedfall — DISMISSED** [review]
Conflig applied on production. Just need this to be rebased so it can be merged.

**Comment by PowerfulBacon** [comment]
Will get round to it

**Comment by PowerfulBacon** [comment]
god I love github

**Review by Crossedfall — APPROVED** [review]
